src/Common/Common.Client/PowerBIGetCmdlet.cs

```diff
@@ -12,7 +12,7 @@ namespace Microsoft.PowerBI.Common.Client
 {
     public abstract class PowerBIGetCmdlet : PowerBIClientCmdlet
     {
-        protected const string AllParameterSetName = "WithAll";
+        protected const string AllParameterSetName = "All";
 
         public PowerBIGetCmdlet(): base() { }
```

Comment:
Don't you also need to update  while (count == top); statement?
Based on what Harjinder said, it sounded like we have to keep trying to get results till we get no more results when filter is used

Reply:
But I could have misunderstood how the API will return the result

Reply:
Discussed offline, the existing changes are good, since on the service side it will fetch all the existing workspaces whenever -Top and -Skip are provided along with the -Filter.